ui/src/templates/api/index.ts

```diff
@@ -229,23 +235,44 @@ const createVariablesFromTemplate = async (
   }
   const variablesIncluded = findIncludedVariables(included)
 
-  const existingVariables = await client.variables.getAll(orgID)
+  const resp = await apiGetVariables({query: {orgID}})
+  if (resp.status !== 200) {
+    throw new Error(resp.data.message)
+  }
+
+  const variables = resp.data.variables.map(v => addVariableDefaults(v))
 
   const variablesToCreate = findVariablesToCreate(
-    existingVariables,
+    variables,
     variablesIncluded
   ).map(v => ({...v.attributes, orgID}))
 
-  const createdVariables = await client.variables.createAll(variablesToCreate)
+  const pendingVariables = variablesToCreate.map(vars =>
+    apiPostVariable({data: vars})
+  )
+
+  const resolvedVariables = await Promise.all(pendingVariables)
+  if (
+    resolvedVariables.length > 0 &&
+    resolvedVariables.every(r => r.status !== 201)
```

Comment:
```suggestion
    resolvedVariables.some(r => r.status !== 201)
```

Reply:
there are a few more examples of this.

Reply:
Totally, I think I was aiming for `every === 201`, good catch!